platform/nodejs/eslintrc.json

```diff
@@ -9,7 +9,20 @@
         "sourceType": "module"
     },
     "rules": {
-        "indent": [ "warn", 4 ],
+        "indent": [
+          "warn",
+          4,
+          {
+            "ArrayExpression": "off",
```

Comment:
If you want to turn any of these on, you'll have to find a way to address the issues.

For example, if you turn on `ArrayExpression` here, you have this in the code:

```js
Parser.netOptionTokenIds = new Map([
    [ '1p', OPTToken1p ],
        [ 'first-party', OPTToken1p ],
    [ 'strict1p', OPTToken1pStrict ],
    [ '3p', OPTToken3p ],
        [ 'third-party', OPTToken3p ],
    [ 'strict3p', OPTToken3pStrict ],
    …
];
```

I have turned these off for now.

Reply:
Would the following fixes the warning?

```js
    [ '1p', OPTToken1p ],
    /* synonym */[ 'first-party', OPTToken1p ],
```

Reply:
>Would the following fix the warning?

Yes, that's exactly what I had done initially (but with an empty comment). It fixes the warning. Let me know if you want to go with this.

Reply:
Yes, I'm fine with this, the original intent is even better fulfilled by being explicit about it.

Reply:
>Let me know if you want to go with this.

Sorry, I see GitHub has the option for letting reviewers make edits. It's set on this one.

Reply:
There's also this in `src/js/static-filtering-parser.js`:

```js
this.reProceduralOperator = new RegExp([
    '^(?:', 
        Array.from(parser.proceduralOperatorTokens.keys()).join('|'),
    ')\\('
].join(''));
```

How would you address this one?

Reply:
>How would you address this one?

I've updated the patch and now there's only one warning left for the above.